### PR TITLE
Onboarding_ArcK8s.csx

### PR DESCRIPTION
**Pull request by khaled-zayed**

### CODE REVIEW

File: data/templates/Detector_ArcK8s.csx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        let startTime = datetime({cxt.StartTime});
+        let endTime = datetime({cxt.EndTime});
+        cluster('CdbSupport').database('Support').<YOUR_TABLE_NAME>
+        | where Timestamp >= startTime and Timestamp <= endTime
+        | <YOUR_QUERY>";
+}
+
+[ArmResourceFilter(provider: "Microsoft.Kubernetes", resourceTypeName: "connectedClusters")]
+[Definition(Id = "<YOUR_DETECTOR_ID>", Name = "", Author = "<YOUR_ALIAS>", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<AzureCosmosDB> cxt, Response res)
```
Comment (ShekharGupta1988) [inline data/templates/Detector_ArcK8s.csx:14]: AzureCosmosDB? Should this is be ARMResource

Comment (khaled-zayed) [reply data/templates/Detector_ArcK8s.csx]: Fixed

File: data/templates/Detector_ArcK8s.csx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+private static string GetQuery(OperationContext<AzureCosmosDB> cxt)
```
Comment (ShekharGupta1988) [inline data/templates/Detector_ArcK8s.csx:1]: forgot here

**Review by ShekharGupta1988 — APPROVED** [review]

File: data/templates/Detector_ArcK8s.csx (hunk 2 above)
Comment (khaled-zayed) [reply data/templates/Detector_ArcK8s.csx]: Fixed

**Pull request merged**